src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -383,28 +389,33 @@ def main(main_args):
     superpmi_src_directory = os.path.join(source_directory, 'src', 'coreclr', 'scripts')
     superpmi_dst_directory = os.path.join(correlation_payload_directory, "superpmi")
     arch = coreclr_args.arch
+    platform_name = coreclr_args.platform.lower()
     helix_source_prefix = "official"
     creator = ""
     ci = True
-    if is_windows:
+    if platform_name == "windows":
         helix_queue = "Windows.10.Arm64" if arch == "arm64" else "Windows.10.Amd64.X86.Rt"
-    else:
+    elif platform_name == "linux":
         if arch == "arm":
             helix_queue = "(Ubuntu.1804.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-bfcd90a-20200121150440"
         elif arch == "arm64":
             helix_queue = "(Ubuntu.1804.Arm64)Ubuntu.1804.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-20210531091519-97d8652"
         else:
             helix_queue = "Ubuntu.1804.Amd64"
+    elif platform_name == "osx":
+        helix_queue = "OSX.1100.ARM64" if arch == "arm64" else "OSX.1014.Amd64"
```

Comment:
You do have helix queue for `OSX_x64` but you aren't running the collection. Is it intentional?

Reply:
Yes, I don't think the macOS/x64 and Linux/x64 collections will be that different. I added `OSX.1014.Amd64` to the file for consistency. I can remove it if you want.